src/pages/Login.jsx

```diff
@@ -11,6 +14,19 @@ class Login extends React.Component {
       disableButton: true,
     };
     this.handleChange = this.handleChange.bind(this);
+    this.fetchToken = this.fetchToken.bind(this);
+  }
+
+  async fetchToken() {
+    const { user, email } = this.state;
+    const { propHandlePlayerData } = this.props;
+    const token = await getToken();
+    const data = { user, email, token };
+    propHandlePlayerData(data);
+
+    localStorage.setItem('player', {
```

Comment:
O Readme diz que 

> É MUITO importante utilizar, neste projeto, React com Redux para controlar a disseminação de informação entre os seus componentes. Por isso, no LocalStrage só é permitido haver três chaves (state, token e ranking) e isso será verificado nos testes!

Então imagino que o certo seria só:

```suggestion
    localStorage.setItem('token', token)
```

Reply:
Bouaa, tô arrumando aqui. Valeu!